api/scripts/daily-digest.ts

```diff
@@ -0,0 +1,125 @@
+import { SMTPClient } from "emailjs";
+import { createEmailWithTemplate } from "../emails/htmlEmailUtils.js";
+import config from "@config";
+import type { MonitoringParams } from "../api/V1/monitoringPage.js";
+import { calculateMonitoringPageCriteria } from "../api/V1/monitoringPage.js";
+import { generateVisits } from "../api/V1/monitoringVisit.js";
+import modelsInit from "@models/index.js";
+import { RecordingType } from "@typedefs/api/consts.js";
+
+(async () => {
+  try {
+    const models = await modelsInit();
+    const users = await models.User.findAll({where: {
+      'settings.emailNotifications.dailyDigest': true
+    }});
+    
+    const templateFilename = "daily-digest.html";
+    const client = new SMTPClient(config.smtpDetails);
```

Comment:
This probably wants to be using the `sendEmail` function, which also handles attaching any images, and handling any send errors.